tests/src/Functional/ContactsDashboardTest.php

```diff
@@ -11,7 +11,7 @@
  */
 class ContactsDashboardTest extends BrowserTestBase {
 
-  public static $modules = ['user', 'contacts', 'views'];
+  public static $modules = ['user', 'views', 'help', 'block'];
 
```

Comment:
Config. We add a help block to a region as part of the theme install. We might be able move the block to 'optional'?